internal/scorecard/alpha/tests/olm.go

```diff
@@ -229,7 +229,7 @@ func loadCSV(bundle registry.Bundle, r scapiv1alpha2.ScorecardTestResult,
 		return r
 	}
 	for _, cr := range crs {
-		r = checkOwnedCSVDescriptors(cr, apiCSV, descriptor, r)
+		r = CheckOwnedCSVDescriptors(cr, apiCSV, descriptor, r)
```

Comment:
Any reason this is being exported in this PR? Doesn't seem related to refactoring tests.